docs/UserGuide.adoc

```diff
@@ -51,7 +51,39 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 
 === Ingredients inventory commands
 
-=== Recipe inventory commands
+=== Recipe commands
+
+Recipe commands are commands that change a particular recipe, which you can indicate using the index of the recipe. +
+Some things you can do are to add and remove ingredients from a recipe's ingredient list. +
+You can also write instructions for your recipes.
+
+==== Add an ingredient
+This command allows you to add ingredients to a recipe.
+Format: `recipe INDEX add i/INGREDIENT q/QUANTITY`
+Examples:
+* `recipe 1 add i/Egg q/2` +
+Adds 2 eggs to the 1st recipe's list of ingredients.
```

Comment:
The documentation shows up as a whole chunk, instead of a list.

Reply:
fixed, thanks